app/src/main/java/org/jellyfin/androidtv/ui/startup/StartupToolbarFragment.kt

```diff
@@ -19,6 +19,11 @@ class StartupToolbarFragment : Fragment() {
 			if (activity is StartupActivity) activity.addServer()
 		}
 
+		binding.manageServers.setOnClickListener {
+			val activity = requireActivity()
+			if (activity is StartupActivity) activity.manageServers()
```

Comment:
```suggestion
			val settingsIntent = Intent(requireActivity(), PreferencesActivity::class.java)
			settingsIntent.putExtra(PreferencesActivity.EXTRA_SCREEN, ManageServersScreen::class.qualifiedName)
			startActivity(settingsIntent)
```

The fragment shouldn't depend on a specific activity (we should fix this later for the `addServer` function too)